src/Dockerfile.dagster-manylinux-builder

```diff
@@ -12,8 +12,8 @@
 #
 # $ echo $GITHUB-PAT | docker login ghcr.io -u USERNAME --password-stdin
 #
-# $ docker build . -f src/Dockerfile.dagster-manylinux-builder -t ghcr.io/dagster-io/dagster-manylinux-builder:latest
-# $ docker push ghcr.io/dagster-io/dagster-manylinux-builder:latest
+# $ docker build . -f src/Dockerfile.dagster-manylinux-builder -t ghcr.io/dagster-io/dagster-manylinux-builder:YOUR_IMAGE_TAG_HERE
```

Comment:
we're now publishing these with a tag that matches the base manylinux version used, [eg](https://github.com/orgs/dagster-io/packages/container/dagster-manylinux-builder/266101571?tag=manylinux_2_28_x86_64).

Should we document that here?

Reply:
Shouldn't we change the tag whenever the underlying image changes, even if the base manylinux version hasn't changed?

Reply:
Otherwise there is no way to downgrade if there is an issue

Reply:
Yeah we could push a tag-per-build but there is no versioning system set up for that.